Code/skyrim_ui/src/app/components/dropdown/dropdown.component.html

```diff
@@ -7,8 +7,7 @@
   <div class="box" (click)="toggle($event)">
     <ng-container>
       <div class="dropdown-selected">
-        <span>{{ options?.[selected] ? options[selected].text : placeholder }}</span>
-        <fa-icon [icon]="faCaretDown"></fa-icon>
+        <span>{{ (options?.[selected] ? options[selected].text : placeholder) | transloco: {length: options?.[selected]?.value} }}</span>
```

Comment:
Because we know pipe the options and the selected value through the transloco function, we now get a bunch of errors for the language dropdown because they don't have translations are already in their respective languages.